src/method.jl

```diff
@@ -95,7 +132,8 @@ function arguments(m::Method)
     end
 end
 
-function where_parameters(x::TypeVar)
+# type-vars can only show up attached to UnionAlls.
+function where_parameter1(x::TypeVar)
```

Comment:
This rename is because it doesn't return  list of where parameters
just formats 1 of them

Reply:
I feel like we need something a bit more descriptive here.

Reply:
I can't think of a better one.
Its not unsual to have:
```
foos(xs) = map(foo1, xs)
```

This is the processing for single thing that kind of maps to the multiple things that `whereparams` is trying to process.

Reply:
> Its not unsual to have:
> `foos(xs) = map(foo1, xs)`

I've never seen this before (in Julia, Python or Scala that i've seen)

Reply:
but i've no better name suggestion - sorry

Reply:
> I've never seen this before (in Julia, Python or Scala that i've seen)

Just a different way of doing things.

```julia
julia> @time [x^2 for x in [1,2,3]]
  0.022936 seconds (41.61 k allocations: 2.128 MiB)
3-element Array{Int64,1}:
 1
 4
 9

julia> @time map( x -> x^2, [1,2,3])
  0.023335 seconds (48.81 k allocations: 2.499 MiB)
3-element Array{Int64,1}:
 1
 4
 9
```

Reply:
I meant for `1` suffix naming convention

Reply:
That makes more sense 😆